src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Logging/Log4netWrapper.cs

```diff
@@ -34,26 +34,16 @@ public AfterWrappedMethodDelegate BeforeWrappedMethod(InstrumentedMethodCall ins
             var xapi = agent.GetExperimentalApi();
             xapi.IncrementLogLinesCount(logLevel);
```

Comment:
Not sure the wrapper should update the log lines separately from recording the log message. Do we want a count of all logs regardless of if we capture them, or a count of those that we do capture?

Reply:
The count should be all log lines the are seen.